dashboard/test/models/levels/gamelab_jr_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class GamelabJrTest < ActiveSupport::TestCase
+  def setup
+    @gamelab_jr = GamelabJr.new
+  end
+
+  test 'uses_google_blockly? returns true when DCDO flag is true' do
+    DCDO.stubs(:get).with('sprite_lab_google_blockly', false).returns(true)
+    assert @gamelab_jr.uses_google_blockly?
+  end
+
+  test 'uses_google_blockly? returns false when DCDO flag is false' do
+    DCDO.stubs(:get).with('sprite_lab_google_blockly', false).returns(false)
+    refute @gamelab_jr.uses_google_blockly?
+  end
+
+  test 'uses_google_blockly? returns false as default' do
```

Comment:
This test will break when we flip the test environment, right? Is that brittle or intended? 🙃 Thoughts @molly-moen?

Reply:
hmm yeah I think we probably don't need this one given we have the above two. Because this will behave differently on drone vs test so there's no way to make this work once we flip on test.

Reply:
That's what I was worried about, thanks!